pkgs/common-updater/scripts/update-source-version

```diff
@@ -229,12 +229,14 @@ if [[ -n "$newUrl" ]]; then
         die "Couldn't evaluate source url from '$attr.$sourceKey'!"
     fi
 
-    # Escape regex metacharacter that are allowed in store path names
```

Comment:
The URL check is there intentionally to detect situation where version changes but update script does not update the URL. Though maybe we could skip the check with `ignoreSameVersion` when explicit URL is provided.

Reply:
> Though maybe we could skip the check with `ignoreSameVersion` when explicit URL is provided.

So `if [[ -z "$ignoreSameVersion" || "$oldUrl" != "$newUrl" ]]; then`?

Reply:
Actually, disregard that as well – your change makes sense as it is. The check I had in mind is the comparison next to version.